content/docs/authorization/deployment/rpm/_index.md

```diff
@@ -19,7 +19,21 @@ The CSM for Authorization proxy server requires a Linux host with the following
 
 These packages need to be installed on the Linux host:
 - container-selinux
-- https://rpm.rancher.io/k3s/stable/common/centos/7/noarch/k3s-selinux-0.4-1.el7.noarch.rpm
+- k3s-selinux-0.4-1
+
+Use the appropriate package manager on the machine to install the packages.
```

Comment:
Can we provide specific instructions for installing on dark sites? yum will not work in that case.

Reply:
Updated. We decided to leave it fairly general.